openai/manifest.json

```diff
@@ -37,6 +37,20 @@
       "Supported OS::Linux",
       "Supported OS::Windows",
       "Supported OS::macOS"
+    ],
+    "resources": [
+      {
+        "resource_type": "blog",
+        "url": "https://www.datadoghq.com/blog/monitor-openai-with-datadog/"
+      },
+      {
+        "resource_type": "blog",
+        "url": "https://www.datadoghq.com/blog/monitor-azure-openai-with-datadog/"
+      },
+      {
+        "resource_type": "product_page",
```

Comment:
would this be other?

Reply:
this is just linking to a recommended monitor type which will live on monitoring resources anyways. we should remove